trunk/source/WitnessParser/src/de/uni_freiburg/informatik/ultimate/witnessparser/WitnessParser.java

```diff
@@ -51,7 +53,7 @@
  */
 public class WitnessParser implements ISource {
 
-	private static final String[] FILE_TYPES = new String[] { "graphml" };
+	private static final String[] FILE_TYPES = new String[] { "graphml", "yaml" };

```

Comment:
The typical file extension for YAML is `.yml`, isn't it?

Reply:
Both are fine, I think. We might need to check the naming for SV-COMP (also for generating).

Reply:
:+1: It seems both are used. Ideally then, we should support both for validation, and fix one for generating witnesses. Currently, the examples in the  sv-witnesses repo use both extensions.